### PR TITLE
linted all python files using black linter

### PR DESCRIPTION
**Pull request by per-sephone**
Used the black linter to format all python files.

### CODE REVIEW

**Review by jasonandmonte — APPROVED** [review]
Hi Nora!

Everything looks good to me!

## Setup
To get black up and running I used `pip install black` and then ran the `black` command on the `src` and `test` directories.

## Notes
One thing we should circle back to in our sync meeting is how we to handle dev dependencies for the project. I think one route is to have a `dev-requirements.txt`, but there is also something like a pipfile or Poetry which could be interesting to implement.

File: src/form.py [hunk 1]
```diff
@@ -99,14 +99,102 @@
...
 GUI_KEY_NAMES = [
-    "C0", "C#0", "D0", "D#0", "E0", "F0", "F#0", "G0", "G#0", "A0", "A#0", "B0",
-    "C1", "C#1", "D1", "D#1", "E1", "F1", "F#1", "G1", "G#1", "A1", "A#1", "B1",
-    "C2", "C#2", "D2", "D#2", "E2", "F2", "F#2", "G2", "G#2", "A2", "A#2", "B2",
-    "C3", "C#3", "D3", "D#3", "E3", "F3", "F#3", "G3", "G#3", "A3", "A#3", "B3",
-    "C4", "C#4", "D4", "D#4", "E4", "F4", "F#4", "G4", "G#4", "A4", "A#4", "B4",
-    "C5", "C#5", "D5", "D#5", "E5", "F5", "F#5", "G5", "G#5", "A5", "A#5", "B5",
-    "C6", "C36", "D6", "D#6", "E6", "F6", "F#6", "G6", "G#6", "A6", "A#6", "B6",
-    "C7", "C#7", "D7" ,"D#7", "E7", "F7", "F#7", "G7", "G#7", "A7", "A#7", "B7"
+    "C0",
```
Comment (kaleb2) [inline src/form.py:102]: This isn't a huge deal, but I actually prefer the original formatting here. Is there a way to have black use the original formatting for long arrays? This formatting is just a lot of extra lines.
Again, not a big deal. Still mergeable, just something to think about.

**Review by kaleb2 — APPROVED** [review]
see minor comment about array formatting.

File: src/form.py (hunk 1 above)
Comment (jasonandmonte) [reply src/form.py:102]: I'm not a huge fan the list formatting either. There is an interesting thread about it here: https://github.com/psf/black/issues/1281

I'm not sure if black has something similar to a `.eslintrc` file where we could modify how it handles long lists.

Comment (per-sephone) [reply src/form.py:102]: I agree with this. I'm going to update this section of the code in my PR #2

**Pull request merged**